setup_test_cache

```diff
@@ -1,9 +1,12 @@
-#! /bin/sh -x
+#! /bin/bash -x
+usage="Usage: setup_test_cache CRDS_TEST_ROOT <c/u> where 'c' means clean and 'u' means update"
+update=${2:-"c"}
+export CRDS_TEST_ROOT=${1?$usage}
 
-export CRDS_TEST_ROOT=$1
-
-(cd $CRDS_TEST_ROOT; rm -rf crds-cache-test;  git clone https://github.com/spacetelescope/crds-cache-test.git;)
-(cd $CRDS_TEST_ROOT; rm -rf crds-cache-default-test; mkdir crds-cache-default-test;)
+if [ "$update" = "c" ]; then
```

Comment:
should be a '==', no?

Reply:
Looks good otherwise.

Reply:
Nope, that is the way the `[` operator works. [Reference here](https://linuxize.com/post/how-to-compare-strings-in-bash/#comparison-operators).  Yes, it is bash and opaque is the best description.

Reply:
Ah interesting, I was looking at [this page](https://ryanstutorials.net/bash-scripting-tutorial/bash-if-statements.php) to make sure and saw `
    #!/bin/bash
    # elif statements
    if [ $1 -ge 18 ]
    then
    echo You may go to the party.
    elif [ $2 == 'yes' ]
    then
    echo You may go to the party but be back before midnight.
    else
    echo You may not go to the party.
    fi
`

as an example. If it works, it works.